packages/manager/src/components/TooltipIcon/TooltipIcon.tsx

```diff
@@ -0,0 +1,102 @@
+import SuccessOutline from '@mui/icons-material/CheckCircleOutlined';
+import ErrorOutline from '@mui/icons-material/ErrorOutline';
+import WarningOutline from '@mui/icons-material/WarningAmberOutlined';
+import InfoOutline from '@mui/icons-material/InfoOutlined';
+import HelpOutline from '@mui/icons-material/HelpOutline';
+import * as React from 'react';
+import IconButton from 'src/components/core/IconButton';
+import Tooltip, { TooltipProps } from 'src/components/core/Tooltip';
+import { SxProps } from '@mui/system';
+import { useTheme } from '@mui/material/styles';
+
+export const TOOLTIP_ICON_STATUS = {
+  SUCCESS: 'success',
+  ERROR: 'error',
+  WARNING: 'warning',
+  INFO: 'info',
+  HELP: 'help',
+} as const;
+
+export type ObjectValues<T> = T[keyof T];
+
+export type TooltipIconStatus = ObjectValues<typeof TOOLTIP_ICON_STATUS>;
+
+interface Props
+  extends Omit<TooltipProps, 'leaveDelay' | 'title' | 'children'> {
+  sx?: SxProps;
+  sxTooltipIcon?: SxProps;
+  text: string | JSX.Element;
+  className?: string;
+  interactive?: boolean;
+  status: TooltipIconStatus;
```

Comment:
What benefit does the `TOOLTIP_ICON_STATUS` object give us compared to just using a string literal for the `status`?

```suggestion
  status: 'success' | 'error' | 'warning' | 'info' | 'help'
```

or 

```typescript
type TooltipStatus = 'success' | 'error' | 'warning' | 'info' | 'help';

interface Props {
  ...
  status: TooltipStatus;
}
```

Reply:
For starters, it provides auto-completion. Also, it prevents someone from passing a random or misspelled `status`.